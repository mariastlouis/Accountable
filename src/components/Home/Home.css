@import url("https://fonts.googleapis.com/css?family=Pathway+Gothic+One");
@import url("https://fonts.googleapis.com/css?family=Cabin");
@import url("https://fonts.googleapis.com/css?family=Raleway:300,400,500,600");
body {
  font-family: 'Raleway', sans-serif;
  color: #545252; }

h1, h2, h3 {
  -webkit-margin-before: 0;
  -webkit-margin-after: 0;
  -webkit-margin-start: 0px;
  -webkit-margin-end: 0px; }

h1 {
  font-size: 40px;
  font-family: 'Raleway', sans-serif;
  font-weight: 400; }

h2 {
  font-family: 'Raleway', sans-serif;
  font-weight: 300;
  display: block; }

p {
  font-size: 16px;
  font-family: "Cabin", sans-serif; }

.main-hed {
  margin-bottom: 20px;
  margin-top: 20px; }

.website-description {
  color: white;
  display: flex;
  align-items: center;
  background: #3a6985;
  flex-direction: column; }

.description-text {
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  width: 95%;
  margin: 10px;
  font-size: 20px; }

.icon {
  font-size: 30px; }
