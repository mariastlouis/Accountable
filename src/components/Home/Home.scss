@import '../../assets/variable.scss';

.main-hed {
  margin-bottom: 20px;
  margin-top: 20px;
}

.website-description {
 color: white;
  display: flex;
  align-items: center; 
  background:$accent-blue;
  flex-direction: column;

}

.description-text {
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  width: 95%;
  margin: 10px;
  font-size: 20px;

}

.icon {
  font-size: 30px;
}