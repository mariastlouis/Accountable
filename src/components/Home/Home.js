import React from 'react';
import AddressForm from '../AddressForm/AddressForm';
import Header from '../../containers/Header/Header';
import './Home.css';
import MapMarker from 'react-icons/lib/fa/map-marker';
import Binoculars from 'react-icons/lib/fa/binoculars';
import Search from 'react-icons/lib/fa/search';

const Home = () => {
  return (
    <div className = 'home'>
      <Header />
      
      <div className = "website-description">
        <div className = "content-container">
          <h1 className = 'main-hed'> Dig deep into Colorado State Politics </h1>
          <div className = "description-text">
            <div> <span className = "icon"> <MapMarker /> </span> Find your state lawmakers</div>
            <div> <span className = "icon"> <Binoculars /> </span> Track their progress </div>
            <div> <span className = "icon"> <Search /> </span>Search bills and committees </div>

          </div>
        </div>
      </div>
      
      <AddressForm />
    </div>  
  );
};

export default Home;