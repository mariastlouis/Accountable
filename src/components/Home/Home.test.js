import  Home  from './Home.js';
import React from 'react';
import { shallow } from 'enzyme';

describe ('Home component tests', () => {
let renderedComponent;

beforeEach (() => {
  renderedComponent = shallow (
    <Home />);
});

  it('should render', () =>{
    expect(renderedComponent).toBeDefined();
  });

  it('should have a main-hed, and description text and a website description', () =>{
    expect(renderedComponent.find('.main-hed').length).toEqual(1);
    expect(renderedComponent.find('.description-text').length).toEqual(1);
    expect(renderedComponent.find('.website-description').length).toEqual(1);
  });

  it('should match the snapshot', () => {
    expect(renderedComponent).toMatchSnapshot();
  });

});