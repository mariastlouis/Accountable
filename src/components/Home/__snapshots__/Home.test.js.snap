// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component tests should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Home />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Header />,
        <div
          className="website-description"
>
          <div
                    className="content-container"
          >
                    <h1
                              className="main-hed"
                    >
                               Dig deep into Colorado State Politics 
                    </h1>
                    <div
                              className="description-text"
                    >
                              <div>
                                         
                                        <span
                                                  className="icon"
                                        >
                                                   
                                                  <FaMapMarker />
                                                   
                                        </span>
                                         Find your state lawmakers
                              </div>
                              <div>
                                         
                                        <span
                                                  className="icon"
                                        >
                                                   
                                                  <FaBinoculars />
                                                   
                                        </span>
                                         Track their progress 
                              </div>
                              <div>
                                         
                                        <span
                                                  className="icon"
                                        >
                                                   
                                                  <FaSearch />
                                                   
                                        </span>
                                        Search bills and committees 
                              </div>
                    </div>
          </div>
</div>,
        <withRouter(Connect(AddressForm)) />,
      ],
      "className": "home",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="content-container"
>
            <h1
                        className="main-hed"
            >
                         Dig deep into Colorado State Politics 
            </h1>
            <div
                        className="description-text"
            >
                        <div>
                                     
                                    <span
                                                className="icon"
                                    >
                                                 
                                                <FaMapMarker />
                                                 
                                    </span>
                                     Find your state lawmakers
                        </div>
                        <div>
                                     
                                    <span
                                                className="icon"
                                    >
                                                 
                                                <FaBinoculars />
                                                 
                                    </span>
                                     Track their progress 
                        </div>
                        <div>
                                     
                                    <span
                                                className="icon"
                                    >
                                                 
                                                <FaSearch />
                                                 
                                    </span>
                                    Search bills and committees 
                        </div>
            </div>
</div>,
          "className": "website-description",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h1
                className="main-hed"
>
                 Dig deep into Colorado State Politics 
</h1>,
              <div
                className="description-text"
>
                <div>
                                 
                                <span
                                                className="icon"
                                >
                                                 
                                                <FaMapMarker />
                                                 
                                </span>
                                 Find your state lawmakers
                </div>
                <div>
                                 
                                <span
                                                className="icon"
                                >
                                                 
                                                <FaBinoculars />
                                                 
                                </span>
                                 Track their progress 
                </div>
                <div>
                                 
                                <span
                                                className="icon"
                                >
                                                 
                                                <FaSearch />
                                                 
                                </span>
                                Search bills and committees 
                </div>
</div>,
            ],
            "className": "content-container",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": " Dig deep into Colorado State Politics ",
                "className": "main-hed",
              },
              "ref": null,
              "rendered": " Dig deep into Colorado State Politics ",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <div>
                     
                    <span
                                        className="icon"
                    >
                                         
                                        <FaMapMarker />
                                         
                    </span>
                     Find your state lawmakers
</div>,
                  <div>
                     
                    <span
                                        className="icon"
                    >
                                         
                                        <FaBinoculars />
                                         
                    </span>
                     Track their progress 
</div>,
                  <div>
                     
                    <span
                                        className="icon"
                    >
                                         
                                        <FaSearch />
                                         
                    </span>
                    Search bills and committees 
</div>,
                ],
                "className": "description-text",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      " ",
                      <span
                        className="icon"
>
                         
                        <FaMapMarker />
                         
</span>,
                      " Find your state lawmakers",
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    " ",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": Array [
                          " ",
                          <FaMapMarker />,
                          " ",
                        ],
                        "className": "icon",
                      },
                      "ref": null,
                      "rendered": Array [
                        " ",
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "function",
                          "props": Object {},
                          "ref": null,
                          "rendered": null,
                          "type": [Function],
                        },
                        " ",
                      ],
                      "type": "span",
                    },
                    " Find your state lawmakers",
                  ],
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      " ",
                      <span
                        className="icon"
>
                         
                        <FaBinoculars />
                         
</span>,
                      " Track their progress ",
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    " ",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": Array [
                          " ",
                          <FaBinoculars />,
                          " ",
                        ],
                        "className": "icon",
                      },
                      "ref": null,
                      "rendered": Array [
                        " ",
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "function",
                          "props": Object {},
                          "ref": null,
                          "rendered": null,
                          "type": [Function],
                        },
                        " ",
                      ],
                      "type": "span",
                    },
                    " Track their progress ",
                  ],
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      " ",
                      <span
                        className="icon"
>
                         
                        <FaSearch />
                         
</span>,
                      "Search bills and committees ",
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    " ",
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": Array [
                          " ",
                          <FaSearch />,
                          " ",
                        ],
                        "className": "icon",
                      },
                      "ref": null,
                      "rendered": Array [
                        " ",
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "function",
                          "props": Object {},
                          "ref": null,
                          "rendered": null,
                          "type": [Function],
                        },
                        " ",
                      ],
                      "type": "span",
                    },
                    "Search bills and committees ",
                  ],
                  "type": "div",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Header />,
          <div
            className="website-description"
>
            <div
                        className="content-container"
            >
                        <h1
                                    className="main-hed"
                        >
                                     Dig deep into Colorado State Politics 
                        </h1>
                        <div
                                    className="description-text"
                        >
                                    <div>
                                                 
                                                <span
                                                            className="icon"
                                                >
                                                             
                                                            <FaMapMarker />
                                                             
                                                </span>
                                                 Find your state lawmakers
                                    </div>
                                    <div>
                                                 
                                                <span
                                                            className="icon"
                                                >
                                                             
                                                            <FaBinoculars />
                                                             
                                                </span>
                                                 Track their progress 
                                    </div>
                                    <div>
                                                 
                                                <span
                                                            className="icon"
                                                >
                                                             
                                                            <FaSearch />
                                                             
                                                </span>
                                                Search bills and committees 
                                    </div>
                        </div>
            </div>
</div>,
          <withRouter(Connect(AddressForm)) />,
        ],
        "className": "home",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="content-container"
>
              <h1
                            className="main-hed"
              >
                             Dig deep into Colorado State Politics 
              </h1>
              <div
                            className="description-text"
              >
                            <div>
                                           
                                          <span
                                                        className="icon"
                                          >
                                                         
                                                        <FaMapMarker />
                                                         
                                          </span>
                                           Find your state lawmakers
                            </div>
                            <div>
                                           
                                          <span
                                                        className="icon"
                                          >
                                                         
                                                        <FaBinoculars />
                                                         
                                          </span>
                                           Track their progress 
                            </div>
                            <div>
                                           
                                          <span
                                                        className="icon"
                                          >
                                                         
                                                        <FaSearch />
                                                         
                                          </span>
                                          Search bills and committees 
                            </div>
              </div>
</div>,
            "className": "website-description",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h1
                  className="main-hed"
>
                   Dig deep into Colorado State Politics 
</h1>,
                <div
                  className="description-text"
>
                  <div>
                                     
                                    <span
                                                      className="icon"
                                    >
                                                       
                                                      <FaMapMarker />
                                                       
                                    </span>
                                     Find your state lawmakers
                  </div>
                  <div>
                                     
                                    <span
                                                      className="icon"
                                    >
                                                       
                                                      <FaBinoculars />
                                                       
                                    </span>
                                     Track their progress 
                  </div>
                  <div>
                                     
                                    <span
                                                      className="icon"
                                    >
                                                       
                                                      <FaSearch />
                                                       
                                    </span>
                                    Search bills and committees 
                  </div>
</div>,
              ],
              "className": "content-container",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": " Dig deep into Colorado State Politics ",
                  "className": "main-hed",
                },
                "ref": null,
                "rendered": " Dig deep into Colorado State Politics ",
                "type": "h1",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <div>
                       
                      <span
                                            className="icon"
                      >
                                             
                                            <FaMapMarker />
                                             
                      </span>
                       Find your state lawmakers
</div>,
                    <div>
                       
                      <span
                                            className="icon"
                      >
                                             
                                            <FaBinoculars />
                                             
                      </span>
                       Track their progress 
</div>,
                    <div>
                       
                      <span
                                            className="icon"
                      >
                                             
                                            <FaSearch />
                                             
                      </span>
                      Search bills and committees 
</div>,
                  ],
                  "className": "description-text",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        " ",
                        <span
                          className="icon"
>
                           
                          <FaMapMarker />
                           
</span>,
                        " Find your state lawmakers",
                      ],
                    },
                    "ref": null,
                    "rendered": Array [
                      " ",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": Array [
                            " ",
                            <FaMapMarker />,
                            " ",
                          ],
                          "className": "icon",
                        },
                        "ref": null,
                        "rendered": Array [
                          " ",
                          Object {
                            "instance": null,
                            "key": undefined,
                            "nodeType": "function",
                            "props": Object {},
                            "ref": null,
                            "rendered": null,
                            "type": [Function],
                          },
                          " ",
                        ],
                        "type": "span",
                      },
                      " Find your state lawmakers",
                    ],
                    "type": "div",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        " ",
                        <span
                          className="icon"
>
                           
                          <FaBinoculars />
                           
</span>,
                        " Track their progress ",
                      ],
                    },
                    "ref": null,
                    "rendered": Array [
                      " ",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": Array [
                            " ",
                            <FaBinoculars />,
                            " ",
                          ],
                          "className": "icon",
                        },
                        "ref": null,
                        "rendered": Array [
                          " ",
                          Object {
                            "instance": null,
                            "key": undefined,
                            "nodeType": "function",
                            "props": Object {},
                            "ref": null,
                            "rendered": null,
                            "type": [Function],
                          },
                          " ",
                        ],
                        "type": "span",
                      },
                      " Track their progress ",
                    ],
                    "type": "div",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        " ",
                        <span
                          className="icon"
>
                           
                          <FaSearch />
                           
</span>,
                        "Search bills and committees ",
                      ],
                    },
                    "ref": null,
                    "rendered": Array [
                      " ",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": Array [
                            " ",
                            <FaSearch />,
                            " ",
                          ],
                          "className": "icon",
                        },
                        "ref": null,
                        "rendered": Array [
                          " ",
                          Object {
                            "instance": null,
                            "key": undefined,
                            "nodeType": "function",
                            "props": Object {},
                            "ref": null,
                            "rendered": null,
                            "type": [Function],
                          },
                          " ",
                        ],
                        "type": "span",
                      },
                      "Search bills and committees ",
                    ],
                    "type": "div",
                  },
                ],
                "type": "div",
              },
            ],
            "type": "div",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
